### PR TITLE
Update branch catalyst-o365-moodle37

### PR DESCRIPTION
**Pull request by ScottVerbeek**
This will update the branch `catalyst-o365-moodle37`. The commit used for this is https://github.com/microsoft/moodle-local_o365/commit/28cde92cd4bc66e25141192a399728ed6e131fe3

### CODE REVIEW

File: classes/feature/usergroups/coursegroups.php [hunk 1]
```diff
@@ -199,12 +199,25 @@ public function create_groups_for_new_courses() {
...
+                        ['type' => 'user', 'moodleid' => $teacherid])) {
+                        $hasowner = true;
+                        break;
+                    }
+                }
+                if ($hasowner) {
+                    try {
+                        $this->create_team($course->id, $groupobjectrec->objectid, $appid);
+                        $this->mtrace('Catalyst change - Resync group membership when creating team. Course #' . $course->id);
+                        $this->resync_group_membership($course->id);
```
Comment (ScottVerbeek) [inline classes/feature/usergroups/coursegroups.php:215]: This was in conflict. L214 and L215

File: classes/feature/usergroups/coursegroups.php [hunk 2]
```diff
@@ -480,7 +493,7 @@ public function resync_group_membership($courseid, $groupobjectid = null, $curre
         // Get list of users enrolled in the course. These are our intended group members.
         $intendedmembers = [];
         $coursecontext = \context_course::instance($courseid);
-        // Only get suspended students here.
+        // Only get non suspended students here.
```
Comment (ScottVerbeek) [inline classes/feature/usergroups/coursegroups.php:496]: This was also a conflict.

File: classes/feature/usergroups/coursegroups.php (hunk 1 above)
Comment (hdagheda) [reply classes/feature/usergroups/coursegroups.php:215]: looks good to me...

File: classes/feature/usergroups/coursegroups.php (hunk 2 above)
Comment (hdagheda) [reply classes/feature/usergroups/coursegroups.php:496]: all good

**Pull request merged**